pwa/src/hooks/wooRequests.ts

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+import { QueryClient, useQuery } from "react-query";
+import APIService from "../apiService/apiService";
+import APIContext from "../apiService/apiContext";
+import { IFiltersContext } from "../context/filters";
+
+export const useWooRequests = (queryClient: QueryClient) => {
```

Comment:
Checkout the feedback left on `resources/wooRequest.ts`. This name adds more complexity, due to the class being called `Verzoeken`.